
##############################  |  Pretrained Setting Complete !!
/home/eiden/anaconda3/envs/eiden/lib/python3.8/site-packages/torchvision/models/_utils.py:208: UserWarning: The parameter 'pretrained' is deprecated since 0.13 and may be removed in the future, please use 'weights' instead.
  warnings.warn(
/home/eiden/anaconda3/envs/eiden/lib/python3.8/site-packages/torchvision/models/_utils.py:223: UserWarning: Arguments other than a weight enum or `None` for 'weights' are deprecated since 0.13 and may be removed in the future. The current behavior is equivalent to passing `weights=VGG19_Weights.IMAGENET1K_V1`. You can also use `weights=VGG19_Weights.DEFAULT` to get the most up-to-date weights.
  warnings.warn(msg)
  0%|          | 0/100 [00:00<?, ?it/s]
  0%|          | 0/253 [00:00<?, ?it/s]
  0%|          | 0/100 [00:01<?, ?it/s]
Traceback (most recent call last):
  File "./run.py", line 12, in <module>
    main()
  File "./run.py", line 9, in main
    trainer.fit()
  File "/home/eiden/eiden/octc-inpainting/src/AOTGAN/train_T.py", line 159, in fit
    self.visualize(epoch = epoch, image = images[0], mask = masks[0], input_image = input_images[0], comp_image= comp_images[0])
  File "/home/eiden/eiden/octc-inpainting/src/AOTGAN/train_T.py", line 225, in visualize
    plt.imshow(plot_img.cpu().detach().numpy(), cmap = 'gray')
  File "/home/eiden/anaconda3/envs/eiden/lib/python3.8/site-packages/matplotlib/pyplot.py", line 2695, in imshow
    __ret = gca().imshow(
  File "/home/eiden/anaconda3/envs/eiden/lib/python3.8/site-packages/matplotlib/__init__.py", line 1446, in inner
    return func(ax, *map(sanitize_sequence, args), **kwargs)
  File "/home/eiden/anaconda3/envs/eiden/lib/python3.8/site-packages/matplotlib/axes/_axes.py", line 5663, in imshow
    im.set_data(X)
  File "/home/eiden/anaconda3/envs/eiden/lib/python3.8/site-packages/matplotlib/image.py", line 710, in set_data
    raise TypeError("Invalid shape {} for image data"
TypeError: Invalid shape (3, 512, 512) for image data